protocol/app/app.go

```diff
@@ -1340,6 +1398,22 @@ func New(
 		)
 	}
 
+	// Vote Extension setup.
+	voteExtensionsHandler := ve.NewVoteExtensionHandler(
+		logger,
+		app.oracleClient,
+		time.Second,
+		currencypair.NewDeltaCurrencyPairStrategy(app.PricesKeeper),
+		compression.NewCompressionVoteExtensionCodec(
+			compression.NewDefaultVoteExtensionCodec(),
+			compression.NewZLibCompressor(),
+		),
+		app.PreBlocker,
+		oracleMetrics,
+	)
+	app.SetExtendVoteHandler(voteExtensionsHandler.ExtendVoteHandler())
+	app.SetVerifyVoteExtensionHandler(voteExtensionsHandler.VerifyVoteExtensionHandler())
+
```

Comment:
nit: we shld probs over-ride the app's Close() fn. to close the oracle-client / prom-server